ui/tests/test-helper.js

```diff
@@ -1,4 +1,4 @@
-import 'npm:core-js';
+import 'core-js';
```

Comment:
what's this?

Reply:
Polyfills that were necessary for Phantom, I believe. I bet it's safe to delete now 🤔 It's also only used in tests, so it's at least not impacting build size or anything.